src/Microsoft.PowerShell.Commands.Utility/commands/utility/GetDateCommand.cs

```diff
@@ -420,6 +424,10 @@ private string UFormatDateString(DateTime dateTime)
                             sb.Append("{0:tt}");
                             break;
 
+                       case 'P':
+                            sb.Append("{0:tt}".ToLowerInvariant());
```

Comment:
It is culture sensitive - seems we can not use `ToLowerInvariant()`.

Reply:
Ah, thanks for the help - I can take that out, as I wasn't sure of the correct position.

Reply:
Yes - tracing it back, [this help article](https://docs.microsoft.com/en-us/dotnet/api/system.globalization.datetimeformatinfo.amdesignator?view=netframework-4.7.2) suggests that the AMDesignator is only in caps, so it may need a lower-level change.  You're absolutely right though, I was only lowercasing the format string, not the result - coffee first, then code :)

Reply:
You could use `StringUtil.Format()` - see examples in the code.